src/shared/install.c

```diff
@@ -1890,6 +1895,9 @@ static int install_info_symlink_wants(
                 q = create_symlink(paths, i->path, path, true, changes, n_changes);
                 if (r == 0)
                         r = q;
+
+                if (!unit_file_exists(scope, paths, dst))
```

Comment:
unit_file_exists() returns negative if something fails and it cannot be determined if the unit file exists or not. i think we should generate this friendly warning only if we are really sure that a the unit file doesn't exist. i.e. compare the result explicitly with `== 0` because 0 is what is returned if the file is missing